bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -130,6 +130,9 @@ class HoverToolView extends InspectTool.View
       vars = {index: i, x: x, y: y, vx: vx, vy: vy, sx: sx, sy: sy, data_x: data_x, data_y: data_y, rx:rx, ry:ry}
       tooltip.add(rx, ry, @_render_tooltips(ds, i, vars))
 
+      vars['renderer'] = renderer
+      @mget('callback')?.execute(vars)
```

Comment:
@fpliger can I clarify "end up calling it many times when you hit many points on a renderer" - do you mean like http://nbviewer.ipython.org/github/birdsarah/bokeh-miscellany/blob/master/notebooks/slider_example/Bubble%20plot.ipynb where there are lots of points on top of each other?